tactics/class_tactics.ml

```diff
@@ -100,7 +100,7 @@ let _ =
 
 let _ =
   declare_bool_option
-    { optdepr  = false;
+    { optdepr  = true; (* remove in 8.8 *)
```

Comment:
@JasonGross I remember seeing this comment for "Typeclasses Limit Intros". Since then I've removed the deprecation notice for this option. But now the comment appears for "Typeclasses Legacy Resolution". This might again be a GitHub bug. @JasonGross can you confirm that your comment was for "Typeclasses Limit Intros"?